src/audio/pipeline.c

```diff
@@ -154,12 +152,11 @@ int pipeline_complete(struct pipeline *p, struct comp_dev *source,
 {
 	struct pipeline_data data;
 
-	trace_pipe_with_ids(p, "pipeline_complete()");
+	pipe_info(p, "pipeline_complete()");
```

Comment:
what's the difference between e.g. pipe_info() and pipe_cl_info()? 

is this the moral equivalent of pr_err("blah") vs. dev_err(dev, "blah")

Reply:
Pretty much, from what I understand, the *_cl_error one is the equivalent of pr_err.

I personally want to have as few of these _cl_* type prints as possible because without hacks for selecting the active component (like the thing @jajanusz is blocking for) I fear we will end up with some rather coarse selections possible for runtime enabling/disabling in the future.

Reply:
@plbossart Yes, _cl_ is device (/object/instance)-less version and *pipe_cl_* was supposed to indicate that there was pipeline trace class included.

Reply:
@paulstelian97 I hope that I already explained that _cl_ version are going to get the driver object, identified by uuid (this one to be introduced soon, along with another proposal for new debug abi) while device version is going to provide that uuid by back ptr to its parent driver as well. So the fine identification will be back on the sof-logger site (no overhead on the dsp side, that would be another compile-time piece of data, nothing new sent over the trace channel).

Reply:
Ah fair, then guess I'll wait. Yes you did mention it, I just forgot (I need more coffee :) ).